lib/typing/ext/parse_test.go

```diff
@@ -82,23 +80,14 @@ func (e *ExtTestSuite) TestParseFromInterfaceDate() {
 
 func (e *ExtTestSuite) TestParseExtendedDateTime_Timestamp() {
 	tsString := "2023-04-24T17:29:05.69944Z"
-	extTime, err := ParseExtendedDateTime(e.ctx, tsString)
+	extTime, err := ParseExtendedDateTime(nil, tsString)
 	assert.NoError(e.T(), err)
 	assert.Equal(e.T(), "2023-04-24T17:29:05.69944Z", extTime.String(""))
 }
 
 func (e *ExtTestSuite) TestParseExtendedDateTime() {
-	ctx := config.InjectSettingsIntoContext(e.ctx, &config.Settings{
-		Config: &config.Config{
-			SharedTransferConfig: config.SharedTransferConfig{
-				// DD-MM-YY
-				AdditionalDateFormats: []string{"02/01/06"},
-			},
-		},
-	})
-
 	dateString := "27/12/82"
-	extTime, err := ParseExtendedDateTime(ctx, dateString)
+	extTime, err := ParseExtendedDateTime([]string{"02/01/06"}, dateString)
```

Comment:
Btw as of go 1.20 `time.DateTime`, `time.DateOnly`, & `time.TimeOnly` are now things: https://pkg.go.dev/time@master#DateTime